docs/content/docs/reference/telemetry.md

```diff
@@ -63,9 +63,9 @@ Keystone collects telemetry information in the form of two different types of da
 We refer to these two different reports, as “device telemetry” and “project telemetry” respectively.
 
 These reports are forwarded to [https://telemetry.keystonejs.com/](https://telemetry.keystonejs.com/), and are reported separately to minimize any correlation between them insofar as the timing and grouping of that data, that an otherwise combined report may have. We are collecting these two reports for different reasons, and thus have no need to associate them.
-We differentiate and record the type and version of reports from the URL used by Keystone.
 
 We additionally record a timestamp of the time that the report is received by the server at [https://telemetry.keystonejs.com](https://telemetry.keystonejs.com/).
+The type of report and the version of the report is determined by the location the report is forwarded to.
```

Comment:
This, when read with the below additions, removes any ambiguity as to how that occurs